python/PiFinder/ui/preview.py

```diff
@@ -169,7 +234,33 @@ def key_enter(self):
         self.command_queues["camera"].put("exp_save")
 
     def key_number(self, number):
-        if number == 0:
-            self.capture_count += 1
-            capture_imagepath = self.capture_prefix + f"_{self.capture_count :0>3}.png"
-            self.command_queues["camera"].put("save:" + capture_imagepath)
+        if self.align_mode:
+            if number == 0:
```

Comment:
hard to discover, what does it even do, set the solve pixel to the center of the screen? Do we want this to be available? Might be confusing.

Reply:
Definitely need some documentation around this, but indeed it resets the solve position /reticle to the center.  The preview screen will show the reticle placed around the solve_pixel so it's a bit more intuitive than the code might indicate.